t/Net/DRI/Protocol/RRI.t

```diff
@@ -48,65 +48,94 @@ $R2 = $E1 . '<tr:transaction><tr:stid>' . $TRID . '</tr:stid><tr:result>success<
 $rc = $dri->contact_check($dri->local_object('contact')->srid('DENIC-12345-BSP'));
 isa_ok($rc, 'Net::DRI::Protocol::ResultStatus');
 is(defined($rc) && $rc->is_success(), 1, 'Contact successfully checked');
-is_string($R1, '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><registry-request xmlns="http://registry.denic.de/global/1.0" xmlns:contact="http://registry.denic.de/contact/1.0"><contact:check><contact:handle>DENIC-12345-BSP</contact:handle></contact:check></registry-request>', 'Check Contact XML correct');
+is_string($R1, '<?xml version="1.0" encoding="UTF-8" standalone="yes"?><registry-request xmlns="http://registry.denic.de/global/3.0" xmlns:contact="http://registry.denic.de/contact/3.0"><contact:check><contact:handle>DENIC-12345-BSP</contact:handle></contact:check></registry-request>', 'Check Contact XML correct');
 is($dri->get_info('exist', 'contact', 'DENIC-12345-BSP'), 0, 'Contact does not exist');
 
 $R2 = $E1 . '<tr:transaction><tr:stid>' . $TRID . '</tr:stid><tr:result>success</tr:result></tr:transaction>' . $E2;
 
 my $c = $dri->local_object('contact');
-$c->srid('DENIC-99990-10240-BSP');
+$c->srid('DENIC-1000002-MAX');
 $c->type('PERSON');
-$c->name('Theobald Tester');
-$c->org('Test-Org');
-$c->street(['Kleiner Dienstweg 17']);
-$c->pc('09538');
-$c->city('Gipsnich');
+$c->name('Max Mustermann');
+$c->org('DENIC eG');
+$c->street(['Abteilung Business Services - dies ist ein Text um die maximale Anzahl von Zeichen darzustellen, die in einem Adressfeld vorkommen können, die Anzahl der Zeichen ist auf zweihundertfünfundfünfzig begrenzt- Kaiserstrasse fünfundsiebzig bis siebenundsiebzig']);
+$c->pc('60329');
+$c->city('Frankfurt am Main');
 $c->cc('DE');
-$c->voice('+49.123456');
-$c->fax('+49.123457');
 $c->email('email@denic.de');
-$c->sip('sip:benutzer@denic.de');
-$c->remarks('Interesting remark');
-$c->disclose({voice =>1});
 
-# Contact create
+# Contact create (based on: https://member.secure.denic.de/index.php?eID=dumpFile&t=f&f=7279&token=c5ea0100484bd1c67f535dc0fef2a7d6e318db53)
```

Comment:
This is not a public link ?

Reply:
Removed member link and substitute by something like: _(based on RRI v3.0 sample)_ instead